cmd/frontend/graphqlbackend/textsearch.go

```diff
@@ -478,6 +478,12 @@ func searchFilesInRepos(ctx context.Context, args *search.TextParameters) (res [
 		}
 	}
 
+	// if there are no indexed repos and this is a structural search
+	// query, there will be no results. Raise a friendly alert.
+	if len(zoektRepos) == 0 && args.PatternInfo.IsStructuralPat {
+		return nil, nil, fmt.Errorf("No indexed repositories for structural search")
```

Comment:
```suggestion
		return nil, nil, errors.New("no indexed repositories for structural search")
```

No format specifiers so errors.New suffices

Go code review style suggests making errors start with a lowercase word